docs/reference/errors-and-warnings/NU1604.md

```diff
@@ -12,6 +12,35 @@ f1_keywords:
 
 # NuGet Warning NU1604
 
+## Missing Package Version
+
+> Project dependency 'PackageA' does not contain an inclusive lower bound. Include a lower bound in the dependency version to ensure consistent restore results.
+
+### Issue
+
+A project dependency doesn't define a version.<br/><br/>This means that restore used the lowest available version. Each restore will float downwards trying to find a lower version that can be used. This means that restore goes online to check all sources each time instead of using the packages that already exist in the user package folder.
+
+### Solution
+
+Find the `PackageReference` item that does not define the `Version` attribute and add it:
+
+For example change from:
+
+> `<PackageReference Include="PackageA" />`
+
+to:
+
+> `<PackageReference Include="PackageA" Version="9.0.0" />`
+
+If the project is using [NuGet's Central Package Management (CPM)](../../consume-packages/Central-Package-Management.md), you need to update the `<PackageVersion />` item in `Directory.Packages.props` and change from:
```

Comment:
With CPM, an issue that I've oftten seen is missing a `ManagePackageVersionsCentrally` and effectively not opting that project into CPM.
Can we add a line for that as a possible solution too?

Reply:
A problem I've frequently experienced is that syntax errors in Directory.Packages.props causes NuGet to silently ignore the file (although I think a `dotnet build` will tell you about the error).

Reply:
Should we add these 2 suggestions in this list?

Reply:
Oh I see it now. It seems resolved as far as I'm concerned.